src/comps/Sidenav.jsx

```diff
@@ -1,33 +1,37 @@
 import React from 'react'
 import { useLocation } from 'react-router-dom';
+import { useNavigate } from "react-router-dom";
 import '../styles/homepage/sidenav.css'
 import { AiOutlineHome } from 'react-icons/ai';
 import { MdOutlineArchive, MdOutlineDelete, MdOutlineLabel } from 'react-icons/md';
 import { BiUser } from 'react-icons/bi';
 import { IoExitOutline } from 'react-icons/io5';
+import { useNote } from '../helpers/context/note-context';
 
-function Sidenav({ hideAside }) {
+function Sidenav( ) {
+  const { utilsState } = useNote()
   
   const location = useLocation();
   const currPath = location.pathname;
+  let navigate = useNavigate();
 
   return (
     <div className='Sidenav'>
-      <div value={hideAside} className={hideAside ? "title_hidden sidenav_container" : "sidenav_container"}>
+      <div className={utilsState.hideAside ? "title_hidden sidenav_container" : "sidenav_container"}>
         <div className="sidenav_top">
-          <div className={currPath === '/Home' ? 'active_menu sidenav_menu' : 'sidenav_menu'}>
+          <div className={currPath === '/' ? 'active_menu sidenav_menu' : 'sidenav_menu'} onClick={() => navigate("/")}>
```

Comment:
Do checkout [NavLink](https://reactrouter.com/docs/en/v6/api#navlink) component from `react-router-dom`, `By default, an active class is added to a <NavLink> component when it is active.` which you can use for styling